scripts/portfolio.js

```diff
@@ -0,0 +1,50 @@
+(function(module) {
+
```

Comment:
I want to take to heart what Brian was explaining today during class, about adding in my own personal content. This file plus portfolioView.js are fairly optimized at this point, but I'm in the process of brainstorming for new ideas to implement.